lisa/operating_system.py

```diff
@@ -131,16 +170,47 @@ def _get_detect_string(cls, node: Any) -> Iterable[str]:
         cmd_result = typed_node.execute(cmd="cat /etc/SuSE-release", no_error_log=True)
         yield get_matched_str(cmd_result.stdout, cls.__suse_release_pattern)
 
+    def _get_os_version(self) -> OsVersion:
+        return self._os_version
+
 
 class Windows(OperatingSystem):
     def __init__(self, node: Any) -> None:
         super().__init__(node, is_posix=False)
+        self._get_os_version
```

Comment:
What this line is for? Should it be removed?

Reply:
> What this line is for? Should it be removed?

No. This is to make sure to set the property at the initialization time.

Reply:
It's a method, and implemented later. This line returns a method reference, but nothing to consume it. So it should be removed.